Description: Generate compressed themes
Author: Stephen Kitt <skitt@debian.org>

--- a/gui/themes/scummtheme.py
+++ b/gui/themes/scummtheme.py
@@ -12,7 +12,7 @@
 		print ("Invalid theme name: " + themeName)
 		return
 
-	zf = zipfile.ZipFile(themeName + ".zip", 'w')
+	zf = zipfile.ZipFile(themeName + ".zip", 'w', zipfile.ZIP_DEFLATED, 9)
 
 	print ("Building '" + themeName + "' theme:")
 	os.chdir(themeName)
